src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -38,8 +41,8 @@
                 Sfjs.renderAjaxRequests();
 
                 /* Handle toolbar-info position */
-                var toolbarBlocks = document.querySelectorAll('.sf-toolbar-block');
-                for (var i = 0; i < toolbarBlocks.length; i += 1) {
+                var toolbarBlocks = [].slice.call(document.querySelectorAll('.sf-toolbar-block'));
+                for (i in toolbarBlocks) {
```

Comment:
I would leave the iteration with incrementor `i`. On some browsers it can iterate on some unexpected properties inherited from prototype